javascript/chronometer.js

```diff
@@ -1,26 +1,63 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+  // ... your code goes here
+ 
+this.currentTime = 0;
+this.intervalId = 0;
+
+  }
+
+
   startClick(callback) {
     // ... your code goes here
+    setInterval(() => {
+      callback();
+      this.currentTime += 1;
+    },1000);
   }
   getMinutes() {
     // ... your code goes here
+
+let minutes = 0;
+let time = this.currentTime;
+if(time >= 60){
+  minutes = (time/ 60);
+}
+return Math.floor(minutes);
+  
   }
   getSeconds() {
     // ... your code goes here
+    let sec = this.currentTime % 60;
+    // let sec = Math.abs(this.currentTime)
+    return sec;
   }
-  twoDigitsNumber() {
+  twoDigitsNumber(input) {
     // ... your code goes here
+    if(input<10)
+    {
+      return "0" + input;
+    }
```

Comment:
check the correct formatting of if conditionals, also some spacing for the condition inside like this:

```
if (input < 10) {
      return "0" + input;
    }
```